quran/api/tests/serializers/test_ayah_serializer.py

```diff
@@ -0,0 +1,12 @@
+from django.test import SimpleTestCase
+
+from quran.api.serializers import AyahSerializer
+
+
+class TestAyahSerializer(SimpleTestCase):
+    def test_fields(self):
+        expected_fileds = (
+            'number',
+            'text',
+        )
+        self.assertCountEqual(AyahSerializer().get_fields().keys(), expected_fileds)
```

Comment:
What's the value in this test? Aren't you just testing DRF, which is unnecessary?
Additionally, once you decided to serialize other fields, this test will fail, which will require you to change it. That's not ideal.